src/extension.ts

```diff
@@ -205,9 +205,15 @@ export async function activate(context: vscode.ExtensionContext) {
         );
         let uniqueBlames = await getBlame(gitAPI, startLine, endLine);
 
+        // Jira
+        const mostRelevantJiraTicket = await getMostRelevantJiraTicket({
+          userEmail: session.account.label,
+          prTitle: sortedPRs[0].title,
```

Comment:
If there is no PR title, should we still call it?

Reply:
This is a very good question. In the new commit I propose the following (would like your input): 

- Send a pr title, wrangle it on the back-end, and search from there
- If there is no PR Title, send an empty String
- In the latter case, send an empty string, which will return no ticket. In this case no Jira ticket will be retrieved. 
- Then on main.js line 66, The control flow doesn't enter the if (message.data.mostRelevantJiraTicket) condition and addMostRelevantJiraTicket( ) isn't called.

Reply:
Ok, so no fallback to commit message?

Reply:
Is it possible to have a commit message, withou having a PR Title?